stable/kubecost/values.yaml

```diff
@@ -152,6 +159,8 @@ cost-analyzer:
         #   mountPath: /etc/config
     alertmanager:
       enabled: true
```

Comment:
You may consider disabling this + pushGateway by default if you aren't using in core integration. @gracedo as FYI

Reply:
We decided to enable alertmanager by default so we can define some rules and alerts which we felt were important for monitoring kubecost. Pushgateway is off by default